content/en-us/luau/strings.md

```diff
@@ -492,3 +492,39 @@ the following:
 		</tr>
 	</tbody>
 </table>
+
+## String Interpolation
+
+In Luau, you can interpolate strings. To declare an interpolated string, put **backticks** (`` ` ``) around the characters:
+
+```lua
+local string1 = `Hello world!`
+print(string1)  --> Hello world!
+```
+
+Create **placeholders** by wrapping a variable with curly brackets (`{` and `}`) inside the string. Placeholders get combined into a single string:
+
+```lua
+local world = "world"
+local string1 = `Hello {world}!`
+print(string1)  --> Hello world!
+```
+
+Placeholders can contain numbers. Math operations are allowed inside placeholders:
+
+```lua
+local number = 1
+local world = "world"
+local string1 = `Hello {world}, {number} time!`
+local string2 = `Hello {world}, {number + 1} times!`
+
+print(string1) --> Hello world, 1 time!
+print(string2) --> Hello world, 2 times!
+```
+
+You can escape curly brackets by using `\{` and `\}` respectively:
+
+```lua
+print(`Hello \{world\}!`) --> Hello {world}!
+```
```

Comment:
```suggestion
```
There's already a header about [escaping strings](https://create.roblox.com/docs/luau/strings#escaping-strings) in this article.